public/welcome/Signup.tsx

```diff
@@ -0,0 +1,229 @@
+            <input
+              type="text"
+              id="handle"
+              name="handle"
+              className="input-box"
+              placeholder="john.bsky.social"
+              onChange={handleValueChange}
+              value={formValues.handle}
+              required
+            />
+          </div>
+
+          <div className="input-container password-container">
+            <label htmlFor="password">Password</label>
+            <div className="password-box">
+              <input
+                type={showPassword?.password ? "text" : "password"}
+                id="password"
+                name="password"
+                className="input-box "
+                placeholder="********"
+                onChange={handleValueChange}
+                value={formValues.password}
+                required
+              />
+              <div className="password-icons">
+                {showPassword?.password ? (
+                  <HiEyeSlash
+                    onClick={() =>
+                      setShowPassword((previous) => ({
+                        ...previous,
+                        password: false,
+                      }))
+                    }
+                  />
+                ) : (
+                  <HiEye
+                    onClick={() =>
+                      setShowPassword((previous) => ({
+                        ...previous,
+                        password: true,
+                      }))
+                    }
+                  />
+                )}
+              </div>
+            </div>
+          </div>
+
+          <div className="input-container password-container">
+            <label htmlFor="confirmPassword">Confirm Password</label>
+            <div className="password-box">
+              <input
+                type={showPassword.confirmPassword ? "text" : "password"}
+                id="confirmPassword"
+                name="confirmPassword"
+                className="input-box "
+                placeholder="********"
+                onChange={handleValueChange}
+                value={formValues.confirmPassword}
+                required
+              />
+              <div className="password-icons">
+                {showPassword.confirmPassword ? (
+                  <HiEyeSlash
+                    onClick={() =>
+                      setShowPassword((previous) => ({
+                        ...previous,
+                        confirmPassword: false,
+                      }))
+                    }
+                  />
+                ) : (
+                  <HiEye
+                    onClick={() =>
+                      setShowPassword((previous) => ({
+                        ...previous,
+                        confirmPassword: true,
+                      }))
+                    }
+                  />
+                )}
+              </div>
+            </div>
+          </div>
+
+          {errorMessage?.length >= 0 && (
```

Comment:
What if we don't use equal to '=' ?

Because of using it it show empty <h5 className="login-msg"></h5> Which makes empty space before Create Account button

Reply:
good point